tox.ini

```diff
@@ -3,30 +3,28 @@ min_version = 4.0
 requires =
     tox-pypi-filter>=0.14
 envlist =
-    py{39,310,311,312}
+    py{310,311,312}
     py312-devdeps
-    py39-oldestdeps
+    py310-oldestdeps
     codestyle
     build_docs
 
 [testenv]
 pypi_filter = https://raw.githubusercontent.com/sunpy/sunpy/main/.test_package_pins.txt
-# Run the tests in a temporary directory to make sure that we don't import
```

Comment:
I am going to flatly refuse to merge this whole PR with these changes. please revert them.

Reply:
They serve no purpose here or in the template.

Reply:
Yes they do. Part of the template is to help people unfamiliar with the package. And either way here isn't the place for these changes.